modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/spi/listener/CTOnDemandUserEventListener.java

```diff
@@ -62,7 +62,11 @@ private void _deleteOnDemandUser(CTCollection ctCollection)
 				_ticketLocalService.deleteTicket(ticket);
 			}
 
-			_userLocalService.deleteUser(ctCollection.getOnDemandUserId());
+			if (_userLocalService.fetchUser(ctCollection.getOnDemandUserId()) !=
```

Comment:
can you assign it to a variable instead and then pass that value into the delete?